v1/openapi/schemas/result.yaml

```diff
@@ -0,0 +1,47 @@
+x-component: 'schemas/result'
```

Comment:
This looks almost identical to schemas/request. Why don't we create a generic schema with all common fields, then inherit them into request and result structures, and expand them with an allOff array?